app/models/skin.rb

```diff
@@ -48,7 +48,7 @@ class Skin < ApplicationRecord
                                   class_name: 'SkinParent', dependent: :destroy, inverse_of: :parent_skin
   has_many :child_skins, through: :skin_children, inverse_of: :parent_skins
 
-  accepts_nested_attributes_for :skin_parents, allow_destroy: true, reject_if: proc { |attrs| attrs[:position].blank? }
+  accepts_nested_attributes_for :skin_parents, allow_destroy: true, reject_if: proc { |attrs| attrs[:position].blank? || attrs[:parent_skin_title].blank? }
```

Comment:
I did get it resolved! It appeared that bundle wasn't installing properly and I had to run init.sh again instead of gem installing it to resolve it

Reply:
Just committed change to check that both parent_skin_id and parent_skin_title are blank alongside position

Reply:
I'm glad you got your environment working!

It looks like the logic on the check is right, but the bracket placement is a little off -- `attrs[:parent_skin_id.blank?]` calls `:parent_skin_id.blank?` (which returns false), so you end up checking whether `attrs[false]` exists instead of checking whether `attrs[:parent_skin_id]` is blank.

Reply:
oh thank you! I was too excited to call the blank check and called it early. fixed now!